packages/dw-react-controls/src/components/PeoplePicker/UserOrGroupPicker/UserOrGroupPicker.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { getStyles as getBasePickerStyles } from "office-ui-fabric-react/lib/components/pickers/BasePicker.styles";
+import { concatStyleSets } from "office-ui-fabric-react/lib/Styling";
+import { IUserOrGroupPickerStyleProps, IUserOrGroupPickerStyles } from "./UserOrGroupPicker.types";
+
+export const getStyles = (props: IUserOrGroupPickerStyleProps): IUserOrGroupPickerStyles => {
+	const { styles: customStyles } = props;
+
+	const basePickerStyles: IUserOrGroupPickerStyles = getBasePickerStyles(props);
+
```

Comment:
Add error state to control (red border, etc).
See TermPicker.styles.ts for reference.